modules/govuk/manifests/node/s_db_admin.pp

```diff
@@ -49,6 +49,11 @@
     require => Apt::Source['gof3r'],
   }
 
+  package { 'redis-tools':
+    ensure  => $ensure,
+    require => Apt::Source['redis-tools'],
```

Comment:
I don't think this is required as redis-tools is part of the normal apt repo.

Reply:
Removed that `require`